mimic/model/ironic_objects.py

```diff
@@ -149,13 +153,14 @@ def detail_json(self):
                 "hardware/interfaces/1/switch_port_id": "Mimic1/10",
                 "hardware/interfaces/0/switch_chassis_id": "mimic-chassis1"
             },
-            "properties": {
+            "properties": self.properties or {
                 "memory_mb": self.memory_mb,
                 "cpu_arch": "amd64",
                 "local_gb": 32,
                 "cpus": 40
             },
-            "driver_info": {
+            "driver": self.driver,
```

Comment:
I think this is fine without a default, as Ironic doesn't have one. If you want to use one, I'd use the "fake" one for testing.